packages/retail-ui-extensions/src/extension-api/navigation-api/navigation-api.ts

```diff
@@ -7,6 +7,9 @@ export interface NavigationApiContent {
   /** The parameters the screen was displayed with */
   params?: any;
 
+  /** Pops the currently presented screen */
+  pop(): void;
```

Comment:
Is there a case where you'd want to pass `params` here?

Reply:
Usually the `pop` function doesn't have `params` here, just a number of screens to `pop` which is something we maybe should add in the future (adding it right now might lead to some weird stuff happening when dismissing the modals).

So for now I would stick to the same API as `react-navigation` (the navigation we use for POS) and when I start investigating a better approach for navigation, we can see if it makes sense to modify this.

Reply:
Makes sense to me.